lib/obj.js

```diff
@@ -653,7 +654,28 @@ function saveMetadata(req, res, next) {
         log.debug({
             options: opts
         }, 'saveMetadata: entered');
-        req.moray.putMetadata(opts, function (err2, obj, data) {
+        /*
```

Comment:
I'm going to need a small education on the state of snaplinks.  I know they don't *work* anymore, but many things create them still (/me glances at SmartOS and Triton release machinery).  Do folks need to be wary of anything after this lands?